src/typings/index.d.ts

```diff
@@ -0,0 +1,101 @@
+export type StakedSNXResponse = {
+	systemStakingPercent: number;
+	timestamp: number;
+	stakedSnx: {
+		ethereum: number;
+		optimism: number;
+	};
+};
+
+export type VolumeData = {
```

Comment:
Another thought, who owns all of the dune dashboards we're grabbing data from? If this they change a query and click save they will break this?

Might be worth thinking of some error handling and maybe runtime validation with zod

Reply:
Yeah I've taken the easy approach on this - just not showing data if there is an issue, but good thoughts if we are investing more on the data side of things